### ISSUE TITLE
Add footer section to home

### ISSUE THREAD

**Issue by STYersonH** [ACM-Chapter-Cusco/kratos#26]
### Motivation

People should be given acces to essential information and links like contact details, legal notices and navigation to important sections because it ensures consistency and accessibility across all web pages

---

### Description

The footer should be created based on the Figma design. It should be done by creating a new directory within components called Footer, where the file index.jsx and possibly other components, if necessary, should be contained. 

```
# Example for the component

├── src\app
│   ├── components
│   │   ├── Footer
│   │   │   ├── index.jsx
│   │   │   ├── ...
└── ...
```

It is also important to consider making the component responsive using TailwindCSS's responsive utilities, something like this
```html
<img class="... md:w-32 lg:w-48">
```

- [x] Code the Footer section
- [x] Design the responsive way for this section
- [x] Code the responsive design for this section

---

### Desing

![Image](https://github.com/user-attachments/assets/5c9ac4f5-28c8-4fad-ac4d-ac075a581e14)

---

### References

- [Design of home page
](https://www.figma.com/design/AAjSxYsBciklIeooXoZd7X/ACM-demo?node-id=200-1504&t=1ix00H4DVBi7UixO-1)
